pkg/skaffold/runner/deployer.go

```diff
@@ -86,7 +92,13 @@ func GetDeployer(ctx context.Context, runCtx *runcontext.RunContext, labeller *l
 				}
 			}
 		}
-
+		if cloudRunDeployFound {
+			if nonHelmDeployFound || len(helmNamespaces) > 0 {
+				// Cloud Run doesn't support multiple deployers in the config.
+				return nil, errors.New("skaffold apply called with both Cloud Run and Kubernetes deployers. Mixing deployment targets is not allowed")
```

Comment:
s/ Mixing deployment targets is not allowed/ Mixing deployment targets is not allowed when using the Cloud Run deployer